### PR TITLE
[QC-873 + QC-874] ITS QC: read geometry file from ccdb, correct reading .json parameters

### PR DESCRIPTION
**Pull request by IsakovAD**
Added optional possibility to read geometry from ccdb

### CODE REVIEW

**Comment by Barthelemy** [comment]
Could you check the O2fst error ? I think that it is due to https://alice.its.cern.ch/jira/browse/QC-874

File: Modules/ITS/src/ITSClusterTask.cxx [hunk 1]
```diff
@@ -97,7 +97,18 @@ void ITSClusterTask::initialize(o2::framework::InitContext& /*ctx*/)
...
+  if (mLocalGeometryFile == 1) {
+    ILOG(Info, Support) << "Getting geometry from local file" << ENDM;
+    o2::base::GeometryManager::loadGeometry(mGeomPath.c_str());
+  } else {
+    ILOG(Info, Support) << "Getting geometry from ccdb - timestamp: " << mGeoTimestamp << ENDM;
+    auto& mgr = o2::ccdb::BasicCCDBManager::instance();
+    mgr.setTimestamp(mGeoTimestamp);
+    mgr.get<TGeoManager>("GLO/Config/GeometryAligned");
+    if (!o2::base::GeometryManager::isGeometryLoaded()) {
+      ILOG(Error, Support) << "Can't retrive geometry from ccdb: " << mgr.getURL() << " timestamp: " << mGeoTimestamp << ENDM;
```
Comment (Barthelemy) [inline Modules/ITS/src/ITSClusterTask.cxx:109]: can it survive such a condition? what will happen later ? should it be a fatal and it exits ?

**Comment by IsakovAD** [comment]
@Barthelemy, @iravasen 

The new commit includes the following updates:

1) .json parameters now obtained with function from utils/common which should provide default values in case of problem with .json + added default values for all parameters
2) Updated handling of error when there is no geomtry in the CCDB

**Review by iravasen — APPROVED** [review]

**Comment by iravasen** [comment]
Thanks a lot @IsakovAD ! Very good. 
@Barthelemy if it looks good also for you, I let you merge the code directly.

**Pull request merged**